src/Raven.Studio/typescript/viewmodels/database/settings/connectionStrings.ts

```diff
@@ -145,17 +145,24 @@ class connectionStrings extends viewModelBase {
         return new getConnectionStringsCommand(this.activeDatabase())
             .execute()
             .done((result: Raven.Client.Documents.Operations.ConnectionStrings.GetConnectionStringsResult) => {
+                // ravenEtl
                 this.ravenEtlConnectionStringsNames(Object.keys(result.RavenConnectionStrings));
+                const groupedRavenEtlNames = _.groupBy(this.ravenEtlConnectionStringsNames(), x => this.hasServerWidePrefix(x));
+                const serverWideNames = _.sortBy(groupedRavenEtlNames.true, x => x.toUpperCase());
+                const regularNames = _.sortBy(groupedRavenEtlNames.false, x => x.toUpperCase());
+                this.ravenEtlConnectionStringsNames(regularNames);
```

Comment:
this.ravenEtlConnectionStringsNames([...regularNames, ...serverWideNames]);

Reply:
by doing that we notify subscribes once

Reply:
done